### ISSUE TITLE
Revisit .editorconfig

### ISSUE THREAD

**Issue by aubricus** [blitzagency/cookiecutter-django#33]
This file has strangely specific config for tabs and spaces, but no default set. Let's make sure this file is up to date.

**Comment by aubricus** [comment]
Suggested edit via @seawolf42 

```
root = true

[*]
charset = utf-8
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
indent_style = space
indent_size = 4

[*.md]
trim_trailing_whitespace = false

[Makefile]
indent_style = tab

[nginx.conf]
indent_size = 2

[*.{yml}]
indent_size = 2```

**Comment by seawolf42** [comment]
fixed by #34
